components/eam/docs/index.md

```diff
@@ -2,6 +2,6 @@
 
 Some introductory text here
```

Comment:
This should be replaced with actual introductory text.  Something like "EAM is a state-of-the-art general circulation model of the atmosphere that ...."

Reply:
Thanks for catching this @rljacob. After consulting with Shaocheng, I've added a couple sentences with a quick overview of EAM.